packages/web-runtime/src/store/spaces.ts

```diff
@@ -57,6 +62,26 @@ const mutations = {
   },
   CLEAR_SPACES(state) {
     state.spaces = []
+  },
+  LOAD_SPACE_MEMBERS(state, members) {
+    state.spaceMembers = members
+  },
+  SPACE_MEMBERS_UPSERT(state, member) {
```

Comment:
Other mutations follow the naming pattern "action first, entity last", i.e. `UPSERT_SPACE_MEMBERS`.